Dockerfile.gpu

```diff
@@ -14,6 +14,7 @@ RUN echo '#!/bin/bash\necho Building container with GPU support' > /usr/bin/nvid
 
 ADD . /gluonts
 
+RUN pip3.7 install mxnet-cu92
```

Comment:
This also be `mxnet-cu92==1.6` or whatever version is specified in the requirements.

Reply:
in the requirements the version is defined as >=1.6.0 so ill use this instead

Reply:
ok, but then why not `mxnet-cu92>=1.6`